### ISSUE TITLE
Equation editor menu bar should stay open if on desktop

### ISSUE THREAD

**Issue by mjc209** [ucam-cl-dtg/isaac-app#744]
Although the change resulting from https://github.com/ucam-cl-dtg/isaac-app/issues/738 was needed to make the equation editor work on smaller screens, on larger screens (most laptops and desktops, I expect) it would be good if the menu bar didn't automatically close - there's space for it and it staying open would make it quicker to put equations together. This is particularly the case for people who want to assemble the equations by first getting all the terms onto the canvas and re-arranging them after that.

In summary:
menu bar should auto-vanish on small screens, but not on large screens

**Comment by Morpheu5** [comment]
I can't reproduce this on live nor on staging. What screen size are you using?

**Comment by mjc209** [comment]
Sorry, it happens purely for the numbers. I'm using 1920x1080
